src/sage/functions/bessel.py

```diff
@@ -1121,8 +1121,8 @@ def Bessel(*args, **kwds):
         sage: f = maxima(Bessel(typ='K')(x,y))
         sage: f.derivative('_SAGE_VAR_x')
         (%pi*csc(%pi*_SAGE_VAR_x) *('diff(bessel_i(-_SAGE_VAR_x,_SAGE_VAR_y),_SAGE_VAR_x,1) -'diff(bessel_i(_SAGE_VAR_x,_SAGE_VAR_y),_SAGE_VAR_x,1))) /2 -%pi*bessel_k(_SAGE_VAR_x,_SAGE_VAR_y)*cot(%pi*_SAGE_VAR_x)
-        sage: f.derivative('_SAGE_VAR_y')
-        -(bessel_k(_SAGE_VAR_x+1,_SAGE_VAR_y)+bessel_k(_SAGE_VAR_x-1, _SAGE_VAR_y))/2
+        sage: f.derivative('_SAGE_VAR_y').sage()
+        -1/2*bessel_K(x + 1, y) - 1/2*bessel_K(x - 1, y)
```

Comment:
I don't think we really care about the exact string output here; we want to know that going to maxima, taking a derivative, and going back to sage gives you the same result that taking the derivative in sage would:

```
sage: x,y = SR.var('x,y')
sage: f = Bessel(typ='K')(x,y)
sage: expected = f.derivative(y)
sage: actual = maxima(f).derivative('_SAGE_VAR_y').sage()
sage: bool(actual == expected)
True
```